perun-core/src/test/java/cz/metacentrum/perun/core/entry/MembersManagerEntryIntegrationTest.java

```diff
@@ -2207,6 +2208,70 @@ public void getAllSponsoredMembers() throws Exception {
 		assertTrue(allSponsoredMembers.contains(richMember2));
 	}
 
+	@Test
+	public void moveMembership() throws Exception {
+		System.out.println(CLASS_NAME + "moveMembership");
+
+		User sourceUser = perun.getUsersManagerBl().getUserByMember(sess, createdMember);
+
+		Member member = setUpMember(createdVo);
+		User targetUser = perun.getUsersManagerBl().getUserByMember(sess, member);
+		// delete member - member was created just to create target user
+		perun.getMembersManager().deleteMember(sess, member);
+		// save user for deletion after test
+		usersForDeletion.add(targetUser);
```

Comment:
I think this is not necessary and it actually doesn't do anything.

Reply:
Removed.